### PR TITLE
LPS-182937 - Add Logic in Table to Sort itens

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

**Review by EugenioAntunes — APPROVED** [review]
Nice job, man 👊🏼 👊🏼 👊🏼

**Review by rafaelfs98 — APPROVED** [review]
LGTM :fire:

**Review by welitonpe — APPROVED** [review]
LGTM :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx [hunk 1]
```diff
@@ -144,19 +144,24 @@ const Table: React.FC<TableProps> = ({
 
 						{columns.map((column, index) => (
 							<ClayTable.Cell headingTitle key={index}>
-								<>
-									{column.value}
+								<div className="d-flex justify-content-between">
+									<span
+										className="cursor-pointer"
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx:149]: No need to add cursor pointer if the column is not sortable, will give a false impression to the user that he can perform an action.

```suggestion
										className={classNames({'cursor-pointer': column.sortable})}
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by kevenleone** [comment]
The rest looks good, thank you @Elvinho

**Comment by kevenleone** [comment]
@Elvinho 

❌ The LPS is missing on your commit message

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**